scala/scala.bzl

```diff
@@ -522,7 +522,7 @@ def _scala_test_impl(ctx):
     (cjars, rjars) = (jars.compiletime, jars.runtime)
     # _scalatest is an http_jar, so its compile jar is run through ijar
     # however, contains macros, so need to handle separately
-    scalatest_jars = _collect_jars([ctx.attr._scalatest]).runtime
+    scalatest_jars = _collect_jars([ctx.attr._scalatest, ctx.attr._scalatest_runner]).runtime
```

Comment:
Yes, this way the added classes in the runner it won't be on the compile-time classpath for all scala_test compiles. 

The scalatest_jars is setup like that to work-around (hack) an issue with java_import forcing ijar creation not meshing well with jars that contain scala macros. Thus, just want to contain how much that hack exposes/gets used.